src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenAsyncTests.vb

```diff
@@ -9184,6 +9184,69 @@ End Module
             CompileAndVerify(compilation, expectedOutput:=expectedOutput)
             CompileAndVerify(compilation.WithOptions(TestOptions.ReleaseExe), expectedOutput:=expectedOutput)
         End Sub
+
+        <Fact>
+        Public Sub GetAwaiterBoxingConversion_01()
+            Dim source =
+"Imports System
+Imports System.Runtime.CompilerServices
+Imports System.Threading.Tasks
+
+Interface IAwaitable
+End Interface
+
+Structure StructAwaitable
+    Implements IAwaitable
+End Structure
+
+Module Program
+    <Extension>
+    Function GetAwaiter(x As IAwaitable) As TaskAwaiter
+        If x Is Nothing Then Throw New ArgumentNullException(Nameof(x))
```

Comment:
>If x Is Nothing Then Throw New ArgumentNullException(Nameof(x)) [](start = 8, length = 63)

Consider printing something so that we can verify that the methods are called.